src/Microsoft.TestPlatform.Common/Utilities/AssemblyResolver.cs

```diff
@@ -175,9 +175,10 @@ private Assembly OnResolve(object sender, AssemblyResolveEventArgs args)
                         }
                     }
                 }
-            }
 
-            return null;
+                this.resolvedAssemblies[args.Name] = null;
```

Comment:
Add EqtTrace.IsInfoEnabled check for main flows.

Reply:
Done